### ISSUE TITLE
Check accessibility on all component pages on site

### ISSUE THREAD

**Issue by jAigret-Bix** [uswds/uswds-site#1104]
- [x] Run accessibility check on all pages
- [x] log findings
- [x] create new issues as needed

**Comment by mejiaj** [comment]
Testing pages in `run-axe.js`:

```js
const PAGES = [
  '/components/',
  '/components/accordion/',
  '/components/alert/',
  '/components/banner/',
  '/components/breadcrumb/',
  '/components/button/',
  '/components/button-groups/',
  '/components/data-visualizations/',
  '/components/footer/',
  '/components/form-controls/',
  '/components/form-templates/',
  '/utilities/layout-grid/',
  '/components/header/',
  '/components/icons/',
  '/components/identifier/',
  '/components/search/',
  '/components/sidenav/',
  '/components/step-indicator/',
  '/components/table/',
  '/components/tag/',
  '/components/tooltip/',
  '/components/typography/',
];
```

## Error results

### Heading order

>id: 'heading-order',
    impact: 'moderate',
    tags: [ 'cat.semantics', 'best-practice' ],
    description: 'Ensures the order of headings is semantically correct',
    help: 'Heading levels should only increase by one',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/heading-order?application=axeAPI',

- Typography
- Tooltip
- Tag
- Table
- Step indicator
- Sidenav
- search
- identifier
- icons
- header
- form-templates
- form-controls
- footer
- data-visualizations
- button-groups
- button
- breadcrumb
- banner
- alert
- accordion

### Landmark unique

>id: 'landmark-unique',
    impact: 'moderate',
    tags: [ 'cat.semantics', 'best-practice' ],
    help: 'Ensures landmarks are unique',
    description: 'Landmarks must have a unique role or role/label/title (i.e. accessible name) combination',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/landmark-unique?application=axeAPI',

- sidenav
- identifier
- footer
- breadcrumb
- banner

### Scope attr valid

>id: 'scope-attr-valid',
    impact: 'moderate',
    tags: [ 'cat.tables', 'best-practice' ],
    description: 'Ensures the scope attribute is used correctly on tables',
    help: 'scope attribute should be used correctly',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/scope-attr-valid?application=axeAPI',

- sidenav
- step indicator
- search
- identifier
- header
- form-controls
- footer
- breadcrumb
- banner
- alert
- accordion

### Color Contrast

>id: 'color-contrast',
    impact: 'serious',
    tags: [ 'cat.color', 'wcag2aa', 'wcag143' ],
    description: 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds',
    help: 'Elements must have sufficient color contrast',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=axeAPI',

- grid

### Scrollable region focusable

>id: 'scrollable-region-focusable',
    impact: 'moderate',
    tags: [ 'wcag2a', 'wcag211' ],
    description: 'Elements that have scrollable content should be accessible by keyboard',
    help: 'Ensure that scrollable region has keyboard access',
    helpUrl: 'https://dequeuniversity.com/rules/axe/3.5/scrollable-region-focusable?application=axeAPI',

- grid

**Comment by thisisdano** [comment]
Fixes will all be in separate PRs.
